lib/Lift/VariableManager.cpp

```diff
@@ -410,6 +411,7 @@ void VariableManager::finalize() {
       Builder.CreateBr(ReturnBB);
 
       // Add the case to the switch
+      // TODO(anjo): TMP
```

Comment:
What's this?

Reply:
No idea TBH xD

Probably I wanted to look at this again so I left a marker in the code, or there was some hacky solution there previously but the comments stuck around.

You can just remove it I think.